Assets/_GameJamCat/Scripts/UI/DossierViewBehaviour.cs

```diff
@@ -34,11 +34,16 @@ public bool IsDossierOpen
             }
         }
 
+        public void SetTargetCat(CatCustomisation catCustomisation)
+        {
+            _dossierMenuView.SetNewCat(catCustomisation);
```

Comment:
might as well null chekc _dossierMenuView (to follow suit with how it is implemented in the Initialize Method)